### PR TITLE
Warnings and bug fixes.

### LINKED ISSUE #1226: Lots of unhandled warnings during test execution
Several of those need to be adressed. The most noise generates this
```
 mslib/msui/_tests/test_mscolab_merge_waypoints.py::Test_Save_Keep_Server_Points::test_save_keep_server_points
  /__w/MSS/MSS/conftest.py:191: UserWarning: An unhandled message box popped up during your test!
  PyQt5.QtWidgets.QMessageBox.critical(): [call(<mslib.msui.mss_pyui.MSSMainWindow object at 0x7f2fcc0e8430>, 'file io plugin error import plugins', "ERROR: Configuration\n\n{'csv': <function load_from_csv at 0x7f300de6f280>}\n\nthrows <class 'ValueError'> error:\n'actionImportFlightTrackcsv' has already been set!"), call(<mslib.msui.mss_pyui.MSSMainWindow object at 0x7f2fcc0e8430>, 'file io plugin error import plugins', "ERROR: Configuration\n\n{'csv': <function save_to_csv at 0x7f300de6f1f0>}\n\nthrows <class 'ValueError'> error:\n'actionExportFlightTrackcsv' has already been set!")]
    warnings.warn(f"An unhandled message box popped up during your test!\n{summary}")
```
which happens in many test cases.

### PR DESCRIPTION
**Pull request by joernu76**
Fix #1226

### CODE REVIEW

File: mslib/mswms/mpl_hsec_styles.py [hunk 1]
```diff
@@ -180,6 +180,11 @@ def _plot_style(self):
         mslp = data['air_pressure_at_sea_level']
 
         # Colors in python2.6/site-packages/matplotlib/colors.py
```
Comment (ReimarBauer) [inline mslib/mswms/mpl_hsec_styles.py]: the comment is likly old

and the prints will make trouble on the server

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
see comments

File: mslib/mswms/mpl_hsec_styles.py (hunk 1 above)
Comment (joernu76) [reply mslib/mswms/mpl_hsec_styles.py]: prints were an oversight. removed

File: mslib/msui/_tests/test_mscolab.py [hunk 2]
```diff
@@ -398,6 +398,9 @@ def test_handle_delete_operation(self, mocktext, mockbox):
         QtWidgets.QApplication.processEvents()
         op_id = self.window.mscolab.get_recent_op_id()
         assert op_id is None
+        QtWidgets.QApplication.processEvents()
+        QtTest.QTest.qWait(0)
```
Comment (ReimarBauer) [inline mslib/msui/_tests/test_mscolab.py:402]: this is 0 ms. Does it makes a difference, without?

**Review by ReimarBauer — APPROVED** [review]
not sure if the qWait with 0ms does something, we can also review all these later

**Comment by joernu76** [comment]
> not sure if the qWait with 0ms does something, we can also review all these later

It shouldn't do anything, but it does. There is an interaction between May catch-all-popup thingie and QT. With the qWait, the window gets destroyed/closed as it should. Without it stays around (maybe invisible). Even a qWait(0) has a quite different effect compared to a processEvents. Argh.

**Pull request merged**